e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js

```diff
@@ -162,4 +162,14 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
 
     cy.url().should('eq', relative('/dashboard/deals/0'));
   });
+
+  it('renders the correct aria-labels based on filter selected', () => {
+    dashboardSubNavigation.deals().invoke('attr', 'aria-label').then((label) => {
+      expect(label).to.equal('deals: ,Filters selected: , Notice Type: , Automatic Inclusion Notice, Manual Inclusion Application, Status: , Draft, Ready for Checker\'s approval');
```

Comment:
what does the `,` character do?

Reply:
I assume it's leaving a slight pause which makes readout a bit clearer

Reply:
ah. never mind. it's for pauses